src/components/Settings/Language/Language.container.js

```diff
@@ -87,10 +83,34 @@ export class LanguageContainer extends Component {
       avaliableAndDownloadablesLangs: [],
       downloadablesOnly: []
     },
-    downloadLangLoading: true,
+    downloadLangLoading: false,
     downloadingLangError: { ttsError: false, langError: false }
   };
 
+  componentDidMount = async () => {
+    const { isdownloading } = this.props.downloadingLang;
+    this.setState({
+      downloadablesLangs: isAndroid()
+        ? this.prepareDownloadablesLenguages()
+        : {
+            avaliableAndDownloadablesLangs: [],
+            downloadablesOnly: []
+          }
+    });
+    if (isdownloading) {
+      this.setState({ downloadLangLoading: true });
+    } else {
+      this.setState({ downloadLangLoading: false });
+    }
```

Comment:
this could be replaced with 
`this.setState({ downloadLangLoading: isdownloading });`

Reply:
done